### ISSUE TITLE
All of the sudden, Aerial seems to have gone wonky

### ISSUE THREAD

**Issue by dsoelter** [JohnCoates/Aerial#1097]
iMac Retina 4K 21.5" 2017. Mojave 10.14.6

Been using Aerial for some time with no issues.  

Recently, I'm getting mostly just black screens with the time at the bottom.  

When the videos do begin to play, they will play only for a short time, then the black screen appears.  

I've tried deleting the screensaver and re-downloading, but to no avail.

Help?

**Comment by glouel** [comment]
Hi @dsoelter 

Can you make a screenshot of your Cache settings ?

**Comment by dsoelter** [comment]
Is either of these helpful?  Bear with me. I’m probably in over my head on this.

**Comment by glouel** [comment]
You need to attach the files from GitHub’s website it doesn’t work by email.

**Comment by dsoelter** [comment]
<img width="753" alt="Screen Shot 2020-09-12 at 11 26 22 AM" src="https://user-images.githubusercontent.com/71188792/93002833-95bff300-f4ee-11ea-9158-16d1e3c19085.png">
<img width="655" alt="Screen Shot 2020-09-12 at 11 27 15 AM" src="https://user-images.githubusercontent.com/71188792/93002835-96588980-f4ee-11ea-951c-8ad907bdf754.png">

**Comment by glouel** [comment]
Thanks a lot @dsoelter. Can you go into Settings > Advanced in Aerial's settings and enable Debug log, like this below : 

<img width="1109" alt="Capture d’écran 2020-09-12 à 21 00 45" src="https://user-images.githubusercontent.com/37544189/93002977-11479180-f53b-11ea-8daf-12d295f306c6.png">

Next time you see a black screen, note the time it happens, and send me the log (you can access it from that place above) so I can look into it.

Thanks and sorry about that.

**Comment by dsoelter** [comment]
[AerialLog.txt](https://github.com/JohnCoates/Aerial/files/5213144/AerialLog.txt)
12:10 Pacific Time - screensaver started with black screen, no video.

**Comment by glouel** [comment]
@dsoelter If I read things correctly:

You launched Aerial in screen saver mode: 

```
2020-09-12 12:09:50.578 : <AerialView: 0x7fc4fdc0a990> AerialView setup init (V2.0.8) preview: false
2020-09-12 12:09:50.578 : Running Version 10.14.6 (Build 18G5033)
```

It initialized correctly, including location.

```
2020-09-12 12:09:51.667 : day
2020-09-12 12:09:51.668 : generate playlist
2020-09-12 12:09:51.668 : playlist count: 119
2020-09-12 12:09:51.668 : Final count : 72
```

It was daytime, Aerial generated a playlsist of 72 videos.

```
20-09-12 12:09:51.669 : <AerialView: 0x7fc4fdc0a990> playing video (OFFLINE MODE) : file:///Users/dalesoelter/Library/Application%20Support/Aerial/Cache/comp_HK_H004_C001_PSNK_DENOISE_v14_HDR_PS_FINAL_20180731_HDR_4K_HEVC.mov
2020-09-12 12:09:51.669 : <AerialView: 0x7fc4fdc0a990> observing current item <AVPlayerItem: 0x6000031aa9e0, asset = <AVURLAsset: 0x6000033c2420, URL = file:///Users/dalesoelter/Library/Application%20Support/Aerial/Cache/comp_HK_H004_C001_PSNK_DENOISE_v14_HDR_PS_FINAL_20180731_HDR_4K_HEVC.mov>>
2020-09-12 12:09:51.669 : <AerialView: 0x7fc4fdc0a990> observeValue Optional("readyForDisplay") false
```

It then started playing a video according to the log. Did you see just a bit of it, or none of it ? This sounds like one of those weird graphic issues, so while it may be unrelated, here's something you could try : 

<img width="1109" alt="Capture d’écran 2020-09-13 à 09 58 34" src="https://user-images.githubusercontent.com/37544189/93013328-c0ba4d80-f5a7-11ea-9473-70aa11dfbfed.png">

This is disabled by default, try enabling it. I've added this because of a weird performance issue on some macs, I think that may be what's causing your issue considering it's a very recent change.

**Comment by dsoelter** [comment]
Tried that just now.  No change.  Here is another log from moments ago.  

Screen most recently is going straight to black, with only the clock showing on the display. Yesterday, at some point I caught a glimpse of one of the videos which played for a few seconds before display went to black, again with only the clock. 

Wondering if it would be helpful to completely remove the application again and start fresh, but would need help knowing where any parts would be hiding after dragging to the trash.
[AerialLog.txt](https://github.com/JohnCoates/Aerial/files/5214676/AerialLog.txt)

**Comment by glouel** [comment]
Thanks a lot, I had another look at your log but it looks fine from there...

I didn't ask, when you go into settings here, do you see a video playing or just the clock too ?

![Capture d’écran 2020-09-13 à 19 14 40](https://user-images.githubusercontent.com/37544189/93024279-88d9f700-f5f5-11ea-9c3f-89c7f717577b.jpg)

Removing should not change a thing, this is a graphics bug so it's more a question of me finding what I changed. I believe that if you tried installing an older version like 2.0.4 you may be fine in the meantime, download here : https://github.com/JohnCoates/Aerial/releases/download/v2.0.4/Aerial.saver.zip

**Comment by dsoelter** [comment]
To answer your question.  I was only seeing black there too>
In the mean time, I think I found the problem.  Video had somehow set itself at HDR, though I have not upgraded to Catalina, and didn't make any changes there.  
Changed to 4K HEVC, which required deleting all videos.  Now, all are downloading again, and I'm seeing what I should be seeing.  Fingers crossed that all is now well.  Thanks a MILLION for all the help.   Never would have solved any of this without your help!
